README.md

```diff
@@ -8,9 +8,19 @@ npm i
 ```
 in the repository root
 
+You also need to install rethinkdb by downloading it and running it from command prompt
+
+Please remember to insert the uservoice details into the config as well as bot token and role ids for Admins/Mods.
+
 # Run
 You can run the bot just as easliy as you can install it, just run
+
+```npm run-script dbcreate
+```
+
```

Comment:
Move this to the install section right under the mention for downloading and install rethinkdb